### DISCUSSION TITLE
Remotely change hot water temperature

### DISCUSSION THREAD

**Discussion by philipgo** [raomin/ESPAltherma#470] [General]
Hello,

has anyone here come up with a solution to remotely change the hot water target temperature? I know that ESPAltherma cannot do this directly, but maybe someone has found a workaround? I can only think of using a fingerbot and I saw someone on Twitter using an ESP32 with servo controlled "finger". 

Any other solutions?

Thank you!

**Comment by Newman820** [comment]
I´m using Daikin LAN-adapter with Home Assistant integration.  That´s the way I can change hot water temperature and climate control directly from HA.

I´m using ESPAltherma because it gives me even more values than Daikin LAN-adapter doesn't deliver.

**Reply by philipgo** [reply]
I think for my 2011 HPSU Compact there is no LAN adapter, unfortunately.

Is there anyone with an older Rotex heat pump who has come up with a solution?

**Reply by DieterB61** [reply]
I think this can only be done by using an additionally smarthome system. 
Do you have a connection to the Daikin Cloud using WiFi?

With my iobroker and the Daikin Cloud Adapter I can switch on and off the the production of hot water but can also not set the setpoint. 

Bou you can set the hot water temperature to the highest  possible value in the heat pump, measure the current temperature using espaltherma and control it by your smart home system by switching the hot water production on or off.

**Reply by DanieleNeo** [reply]
Hi @philipgo, I have no answer to your request, I read that the serial interface has limitations on what can be done (although maybe something can be done), and that there are other projects that use other approaches (P1P2 and CAN bus). It would be nice if the efforts converged towards a common solution.

I read that, like me, you have an old heat pump, could you tell me some definitions you use? Because I find references to a definition that has been removed and I can't figure out which one replaced it... at the moment I use the default but I noticed that some values ​​are missing that don't allow me to calculate the COP (for example)

I also opened an adhoc discussion
https://github.com/raomin/ESPAltherma/discussions/481

D.
